provisioning/playbooks/couchbase/cluster_init.yml

```diff
@@ -0,0 +1,29 @@
+---
+- hosts: couchbase
+  sudo: False
+  gather_facts: False
+  tasks:
+    - name: Create cluster grouping
+      group_by: key={{ couchbase_server_node_role }}
+
+- hosts: primary
+  sudo: True
+#  serial: 1
```

Comment:
I copied it from another task I had with multiple machines. 

> By default, Ansible will try to manage all of the machines referenced in a play in parallel ... you can define how many hosts Ansible should manage at a single time by using the ‘’serial’’ keyword
http://docs.ansible.com/ansible/playbooks_delegation.html#rolling-update-batch-size